src/js/shield_defs.js

```diff
@@ -4153,5 +4155,24 @@ export function loadShields() {
     "Westpark Tollway": "WPT",
   };
 
+  shields["GLCT"].overrideByRef = {
+    LECT: {
+      spriteBlank: "shield_glct_lect",
+      notext: true,
```

Comment:
This is fine, but I _think_ it should be possible to factor out `notext` into the parent object.